.github/vale/styles/Klaw/branding.yml

```diff
@@ -23,3 +23,4 @@ swap:
   "node[ -]?js?": NodeJS
   "type[ -]?scripts?": TypeScript
   "(?i)ssl": SSL
+  "(?i)kafka connect": Kafka Connect
```

Comment:
Could you replace this rule with `"(?i)kafka[ -]?connect": Kafka Connect` and run the checks again? That was my bad, I didn't expect us to have instances where it's written "KafkaConnect". The updated rule will catch those instances, too.